### PR TITLE
Works branch - Portfolio: mobile version

### PR DESCRIPTION
**Pull request by rominana**
Portfolio: mobile version

**Built With**
Html
Css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,36 +1,233 @@
...
+<!--END OF WORK SECTIONS-->
+<!--ABOUT ME SECTION-->
+<section id="about-me">
+    <div class="contenedor">
+      <div class="about-me-content">
+      <h3 class="title"> About Myself</h3>
+      <p class="description"> I'm a software developer! I can help you build a product ,feature or website. Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <p class="connect">LETS CONNECT</p>
+        <div class="social-network">
+            <img src="images/twitter-logo.svg" alt="twitter" />
```
Comment (danielufeli) [inline index.html:173]: - [ ] Nicely done, following the requirement kindly list the social icons using `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag)

File: index.html [hunk 2]
```diff
@@ -1,36 +1,233 @@
...
+          </div>
+          <div class="project-button">
+            <button class="btn" type="button">Get my resume</button>
+          </div>
+        </div>
+      </div>
+  </section>
+<!--BEGGINING OF LANGUAGES-->
+<!--codigo de w3school-->
+<div class="languages">
```
Comment (danielufeli) [inline index.html:187]: - [ ] Great work you have done with your about section, kindly re-align the section so it aligns with what you have in the Figma Template.
### Your Design
skill

### Figma Template
skill-f

File: index.html [hunk 3]
```diff
@@ -1,36 +1,233 @@
...
+          <img src="images/Vector2.svg" alt="#"/>
+          <img src="images/Vector.svg" alt="#"/>
+        </div>
+    </article>
+  </div>
+</main>
+<!--END OF ID SECTION-->
+
+<!--WORK SECTIONS-->
+<section id="work">
```
Comment (danielufeli) [inline index.html:63]: - [ ] Great work you have done with the work section, kindly add a bottom margin to the cards, so it aligns with the Figma Design

### Your Design!
m2\

### Template
f2

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @rominana, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Clean Design ✔️
- Passing linters ✔️
- Descriptive Readme ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rominana** [comment]
Thank you very much for the feedback @danielufeli! All the changes have been made :smile:

File: index.html [hunk 4]
```diff
@@ -1,36 +1,235 @@
...
+          </div>
+          <div class="project-button">
+            <button class="btn" type="button">Get my resume</button>
+          </div>
+        </div>
+      </div>
+  </section>
+<!--BEGGINING OF LANGUAGES-->
+<!--codigo de w3school-->
+<div class="languages">
```
Comment (Ismailco) [inline index.html:189]: - [ ] [OPTIONAL] Greate work implementing the changes 👍🏼 however you may consider removing the little margin on to of the Language so your design could be identical to Figma design, and also please consider adding some margin to the languages cards ```JavaScript```, ```HTML```, ```CSS```

|Your design|Figma design|
|-|-|
|Screenshot_20220528_173001|Screenshot_20220528_173021|

**Review by Ismailco — APPROVED** [review]
### Hi @rominana,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (rominana) [reply index.html:189]: I'll apply those changes. Thanks for the feedback @ismailco :smile:

**Pull request merged**